app/assets/javascripts/discourse/templates/components/queued-post.hbs

```diff
@@ -20,24 +20,41 @@
     </div>
     <div class='clearfix'></div>
 
-    <span class='post-title'>
-      {{i18n "queue.topic"}}
-      {{#if post.topic}}
-        {{topic-link post.topic}}
-      {{else}}
-        {{post.post_options.title}}
-      {{/if}}
-      {{category-badge post.category}}
-    </span>
+    {{#if editTitleAndCategory}}
+      <span class="edit-title">
+        {{text-field value=buffered.title maxlength=siteSettings.max_topic_title_length}}
+      </span>
+      {{category-chooser value=buffered.category_id}}
+    {{else}}
+      <span class='post-title'>
+        {{i18n "queue.topic"}}
+        {{#if post.topic}}
+          {{topic-link post.topic}}
+        {{else}}
+          {{editables.title}}
+        {{/if}}
+        {{category-badge editables.category}}
+      </span>
+    {{/if}}
 
     <div class='body'>
       {{#if editing}}
         {{d-editor value=buffered.raw}}
       {{else}}
-        {{cook-text post.raw}}
+        {{cook-text editables.raw}}
       {{/if}}
     </div>
 
+    {{#if showTags}}
+      <div class="list-tags">
+        {{#each tags as |t|}}
+          {{discourse-tag t}}
+        {{/each}}
+      </div>
+    {{else if editTags}}
+      {{tag-chooser tags=buffered.tags categoryId=buffered.category_id width='100%'}}
```

Comment:
I thought this is the intended way to [set the width on the tag chooser `select2` container](https://github.com/discourse/discourse/blob/130ae2cd4b6d60accde8267f51b0442ccd3217a5/app/assets/javascripts/discourse/components/tag-chooser.js.es6#L61)?

Without setting the width manually, the tag chooser is about 10px wider than everything else. `select2` internally sets the width with an inline style, which I cannot override without an `!important`.  /facepalm

p.s. The `100%` is wrapped by the parent `div.cooked`, and this is [set in the CSS.](https://github.com/discourse/discourse/blob/1146772debf861c5c8b6e4e3cebdba31a1138922/app/assets/stylesheets/desktop/queued-posts.scss#L18)